src/ui/GameRoot.tsx

```diff
@@ -354,7 +354,9 @@ export function GameRoot(): React.ReactElement {
           forceKill={killAllScripts}
           softReset={softReset}
           reactivateTutorial={() => {
-            prestigeAugmentation();
+            prestigeWorkerScripts();
+            Player.getHomeComputer().pushProgram(CompletedProgramName.nuke);
+            Player.currentServer = SpecialServers.Home;
```

Comment:
```suggestion
            Terminal.connectToServer(SpecialServers.Home);
```
Don't change the current server like this. `Terminal.connectToServer` takes care of the necessary steps.

Reply:
Done thanks!

Reply:
Also adding the missing terminal import to fix the build.